packages/common/src/utils/email.ts

```diff
@@ -3,11 +3,11 @@ export const getEmailDomain = (email: string): string => {
         throw new Error(`Invalid email, contains whitespace: ${email}`);
     }
 
-    const domain = email.split('@')[1];
-    if (!domain) {
+    const domains = email.split('@');
+    if (domains.length !== 2) {
```

Comment:
The new code doesn't check if there is a domain, so this new code would accept "user@" as a valid email. 
Perhaps we add `|| !domains[1]`  here to ensure this case is covered, like in the previous version.